docs/making-requests.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Making Requests
```

Comment:
This page used to be `http-requests` but when this gets resynced when its merged into `main` I'll go into the Dash and delete the old `http-requests` page.